data.sql

```diff
@@ -4,4 +4,23 @@ INSERT INTO animals (name, date_of_birth, escape_attempts,neutered,weight_kg) VA
+
+/* Vet clinic database: add "join table" for visits */
+-- Insert the following data for vets:
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23'), ('Maisy Smith', 26, '2019-01-17'), ('Stephanie Mendez', 64, '1981-05-04'), ('Jack Harkness', 38, '2008-06-08');
+-- Insert the following data for specialties:
+INSERT INTO specializations (vet_id, species_id) SELECT vets.id, species.id FROM vets JOIN species ON species.name = 'Pokemon' OR species.name = 'Digimon' WHERE vets.name IN ('William Tatcher', 'Stephanie Mendez', 'Jack Harkness');
+-- Insert the following data for visits:
+INSERT INTO visits (animal_id, vet_id, date) VALUES ((SELECT id FROM animals WHERE name='Agumon'),(SELECT id FROM vets WHERE name='William Tatcher'),'2020-05-24'), ((SELECT id FROM animals WHERE name='Agumon'),(SELECT id FROM vets WHERE name='Stephanie Mendez'),'2020-06-22'), ((SELECT id FROM animals WHERE name='Gabumon'),(SELECT id FROM vets WHERE name='Jack Harkness'),'2021-02-02'), ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2020-01-05'), ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2020-03-08'), ((SELECT id FROM animals WHERE name='Pikachu'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2020-05-14'), ((SELECT id FROM animals WHERE name='Devimon'),(SELECT id FROM vets WHERE name='Stephanie Mendez'),'2021-05-04'), ((SELECT id FROM animals WHERE name='Charmander'),(SELECT id FROM vets WHERE name='Jack Harkness'),'2021-02-24'), ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2019-12-21'), ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name='William Tatcher'),'2020-08-10'), ((SELECT id FROM animals WHERE name='Plantmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2021-04-07'), ((SELECT id FROM animals WHERE name='Squirtle'),(SELECT id FROM vets WHERE name='Stephanie Mendez'),'2019-09-29'), ((SELECT id FROM animals WHERE name='Angemon'),(SELECT id FROM vets WHERE name='Jack Harkness'),'2020-08-03'), ((SELECT id FROM animals WHERE name='Angemon'),(SELECT id FROM vets WHERE name='Jack Harkness'),'2020-11-04'), ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2019-01-24'), ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2019-05-15'), ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2020-02-27'), ((SELECT id FROM animals WHERE name='Boarmon'),(SELECT id FROM vets WHERE name='Maisy Smith'),'2021-08-03'), ((SELECT id FROM animals WHERE name='Blossom'),(SELECT id FROM vets WHERE name='Stephanie Mendez'),'2020-05-24'), ((SELECT id FROM animals WHERE name='Blossom'),(SELECT id FROM vets WHERE name='William Tatcher'),'2021-01-11');
```

Comment:
**[ OPTIONAL]** 
- You can consider making each **SELECT** statement on a new line to make your codebase readable.